iferr/errors.go

```diff
@@ -87,3 +87,27 @@ func (ie *IfErr) WriteHTTP(err error, w http.ResponseWriter, codeArg ...int) boo
 
 	return false
 }
+
+type Opts struct {
+	Code int
+	Message string
+}
+
+func (ie *IfErr) Respond(err error, w http.ResponseWriter, opts ...Opts) bool {
```

Comment:
There should be a package level function Respond as well, so that you can do iferr.Respond() using the default iferr. Look at WriteHTTP for example, there are two WriteHTTP functions.